apps/site/src/lib/admin/authorization.ts

```diff
@@ -16,3 +23,7 @@ export function isAdminRole(role: string | null) {
 export function isCheckinLead(role: string | null) {
 	return role !== null && CHECKIN_ROLES.includes(role);
 }
+
+export function isNonHacker(role: string | null) {
+	return role !== null && NONHACKER_ROLES.includes(role);
+}
```

Comment:
Nitpick [non-blocking]: while this is related to roles, it's not really related to authorization. At the moment, not sure where else to put this, maybe just add a comment suggesting to refactor elsewhere later.